api.csproj

```diff
@@ -4,8 +4,12 @@
     <AzureFunctionsVersion>v3</AzureFunctionsVersion>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.0.0" />
     <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="3.0.*" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.*" />
+    <PackageReference Include="newtonsoft.json" Version="12.0.3" />
```

Comment:
Let's not use Json.NET anymore. .NETCore has its own new thing in `System.Text.Json` which is tunned for performance and was actually mostly done (I think) by the same James Newton-King ;)